CryptoPrices/Sources/MyCoin/Sources/MyCoin/TimeFrameSectionView/TimeFrameSection.swift

```diff
@@ -10,7 +10,7 @@ import SwiftUI
 
 struct TimeFrameSection: View {
     
-    @State private var selected: TimeFrameItem = .init(timeFrame: .oneDay)
+    @Binding var selected: TimeFrameItem
```

Comment:
One thing I just noticed, we actually can use `TimeFilter` directly here, there's no need for the `TimeFrameItem` to exist, which wraps `timeFrame: TimeFilter` without anything else. But it's fine to keep this as is though 🙏

Reply:
Thanks for the suggestion 🙏